contrib/k8s.io/client-go/kubernetes/kubernetes.go

```diff
@@ -21,11 +22,18 @@ func WrapRoundTripper(rt http.RoundTripper) http.RoundTripper {
 	return httptrace.WrapRoundTripper(rt,
 		httptrace.WithBefore(func(req *http.Request, span ddtrace.Span) {
 			span.SetTag(ext.ServiceName, "kubernetes")
-			span.SetTag(ext.ResourceName, requestToResource(req.Method, req.URL.Path))
+			span.SetTag(ext.ResourceName, RequestToResource(req.Method, req.URL.Path))
+			traceID := span.Context().TraceID()
+			if traceID == 0 {
```

Comment:
Why this check?

Reply:
Interesting point. I don't think we've ever considered covering for this case in any of our implementations. Let's leave it then but add a comment:
```suggestion
			if traceID == 0 {
			    // tracer is not running
```